apps/async-worker/async_worker/services/enrichment.py

```
@@ -0,0 +1,66 @@
+                celery_app=celery_app,
+                timeout=DOWNLOAD_VIDEO_TIMEOUT,
+            )
+            output_filename = await pdf_export.export()
+            metadata_item["media_files"].append(
+                {
+                    "media_type": "document",
+                    "url": output_filename,
+                    "caption": "",
+                }
+            )
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)